### DISCUSSION TITLE
How to catch error from login()?

### DISCUSSION THREAD

**Discussion by urbgimtam** [Intevel/nuxt-directus#63] [Q&A] [answered]
I'm sure this is something simple, but I can't seem to uncover it.

I'm trying to print out the error when a login fails.

I can see the error on devtools, as the XHR answer. But I can't seem to get it programatically.

Any thoughts?

(code from [DirectusAuth](https://nuxt-directus.netlify.app/usage/useDirectusAuth) example page)

```
<script setup lang="ts">
const loginData = reactive({
	formEmail: "",
	formPass: "",
})

const { login } = useDirectusAuth()
const router = useRouter()

const onSubmit = async () => {
	console.log("starting login")
	try {
		await login({
			email: loginData.formEmail,
			password: loginData.formPass,
		})
	} catch (e) {
           console.log(e)
       }
}
</script>
```

**Comment by theo-stocchetti** [comment, marked as answer]
This is a feature that is not yet implemented: https://github.com/Intevel/nuxt-directus/issues/75
